tasks/website/src/linter/snapshots/schema_markdown.snap

```diff
@@ -61,7 +61,9 @@ See [Oxlint Rules](./rules)
 
 type: `object`
 
-Shared settings for plugins
+Configure the behavior of linter plugins.
+
+## Example Here's an example if you're using Next.js in a monorepo: ```jsonc { "settings": { "next": { "rootDir": "apps/dashboard/" }, "react": { "linkComponents": [ // support next/link { "name": "Link", "linkAttribute": "to" } ] }, "jsx-a11y": { "components": { "Link": "a", "Button": "button" } } } } ```
```

Comment:
fixme

Reply:
This is problematic, it looks like JSONSchema does not preserve newlines. This is going to make supporting comments in JSON examples difficult.